core/server/data/migrations/versions/4.7/03-add-labs-setting.js

```diff
@@ -0,0 +1,44 @@
+const ObjectID = require('bson-objectid');
+const logging = require('../../../../../shared/logging');
+const {createTransactionalMigration} = require('../../utils.js');
+
+const MIGRATION_USER = 1;
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const labsExists = await knex('settings')
+            .where('key', '=', 'labs')
+            .first();
+
+        logging.info(labsExists);
```

Comment:
I don't think we should leave this here, the migration output should be easily readable

Reply:
This is deffo a debugging leftover! Gonna :fire: now. Thanks for spotting!